server/app/assets/javascripts/api_users.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
I think it should be ok to delete the coffee and scss files that are created in this review from this folder. Can you please check?